libs/@guardian/react-crossword/src/components/AnagramHelper.tsx

```diff
@@ -1,112 +1,71 @@
 import { css } from '@emotion/react';
 import { space } from '@guardian/source/foundations';
-import { SvgCross } from '@guardian/source/react-components';
+import { SvgCross, TextInput } from '@guardian/source/react-components';
 import { useCallback, useEffect, useState } from 'react';
 import { useCurrentClue } from '../context/CurrentClue';
 import { useData } from '../context/Data';
 import { useProgress } from '../context/Progress';
 import { useTheme } from '../context/Theme';
 import { useUIState } from '../context/UI';
-import { useUpdateCell } from '../hooks/useUpdateCell';
-import type { AnagramHelperProgress } from '../utils/getAnagramHelperProgressForGroup';
-import { getAnagramHelperProgressForGroup } from '../utils/getAnagramHelperProgressForGroup';
+import { biasedShuffle } from '../utils/biasedShuffle';
+import { getCellsWithProgressForGroup } from '../utils/getCellsWithProgressForGroup';
 import { Button } from './Button';
 import { Clue } from './Clue';
 import { SolutionDisplay } from './SolutionDisplay';
-import { SolutionDisplayKey } from './SolutionDisplayKey';
 import { WordWheel } from './WordWheel';
 
 export const AnagramHelper = () => {
-	const [shuffled, setShuffled] = useState<boolean>(false);
-	const [candidateLetters, setCandidateLetters] = useState<string[]>([]);
-	const [wordWheelLetters, setWordWheelLetters] = useState<string[]>([]);
-	const [progressLetters, setProgressLetters] = useState<
-		AnagramHelperProgress[]
-	>([]);
-	const { entries } = useData();
-	const { progress } = useProgress();
-	const { updateCell } = useUpdateCell();
+	const [letters, setLetters] = useState<string>('');
```

Comment:
if you set a default value, i don't think you need to type it? not a blocker though